.github/workflows/spawn-containers.yml

```diff
@@ -0,0 +1,16 @@
+name: Spawn Docker containers
+
+on:
+  push:
+    branches:
+      - main
```

Comment:
Push on commit to `main` normally makes great sense (as we discussed), but makes it difficult to test here. You could consider temporarily triggering this flow on pull requests as well.

Reply:
Found the answer myself :grimacing: